scalding-core/src/test/scala/com/twitter/scalding/FileSourceTest.scala

```diff
@@ -216,13 +216,8 @@ class FileSourceTest extends WordSpec with Matchers {
       pathIsGood("test_data/2013/04/") shouldBe false
     }
 
-    "reject an empty directory" in {
-      pathIsGood("test_data/2013/05/") shouldBe false
-      pathIsGood("test_data/2013/05/*") shouldBe false
-    }
-
-    "reject a directory with only _SUCCESS when specified as a glob" in {
-      pathIsGood("test_data/2013/06/*") shouldBe false
+    "accept a directory with only _SUCCESS when specified as a glob" in {
```

Comment:
this is a bit wild. So we are explicitly changing the result of a function that we had tested.

One wonders if anyone will be hit by this. I think that empty with _SUCCESS should be okay, I think. Without _SUCCESS, I would say no.